cloud/linode/vpc_test.go

```diff
@@ -0,0 +1,153 @@
+package linode
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/linode/linode-cloud-controller-manager/cloud/linode/client/mocks"
+	"github.com/linode/linodego"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetAllVPCIDs(t *testing.T) {
+	tests := []struct {
+		name   string
+		vpcIDs map[string]int
+		want   []int
+	}{
+		{
+			name:   "multiple vpcs present",
+			vpcIDs: map[string]int{"test1": 1, "test2": 2, "test3": 3},
+			want:   []int{1, 2, 3},
+		},
+		{
+			name:   "no vpc present",
+			vpcIDs: map[string]int{},
+			want:   []int{},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			vpcIDs = tt.vpcIDs
+			got := GetAllVPCIDs()
+			sort.Ints(got)
+			if !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("GetAllVPCIDs() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestGetVPCID(t *testing.T) {
```

Comment:
qq: any reason to not use tabular test format for this?

Reply:
`ListVPCs()` call and its response is changing in each test run, hence I avoided tabular format. Not sure if we can achieve that with tabular format unless we do if/else with bunch of conditions to decide which mock call to use.

Reply:
Ah i see. I'm okay with that